### ISSUE TITLE
Minor bugs in Keras Demo - Multiclass.ipynb

### ISSUE THREAD

**Issue by hashb** [anniecherk/scikit_learn_and_keras_demo_fall_2017#1]
Looks like someone forgot to convert validation set lablels

```python
# convert class vectors to binary class matrices
y_train = keras.utils.to_categorical(y_train, num_classes)
y_test  = keras.utils.to_categorical(y_test,  num_classes)
y_validation = keras.utils.to_categorical(y_validation, num_classes)
```

Also, I had some trouble with graphviz on macos. I was able to fix it by `brew install graphviz`.
